frontend/src/metabase/visualizations/lib/settings/typed-utils.unit.spec.ts

```diff
@@ -146,4 +151,24 @@ describe("extendCardWithDashcardSettings", () => {
 
     expect(result.visualization_settings).toEqual({ foo: "bar" });
   });
+
+  it("should omit settings that are hidden on dashboards (metabase#61112)", () => {
```

Comment:
With the "Add series modal" + dashcard viz settings gone on cartesian charts it is not possible to recreate this state in the app anymore